src/Microsoft.OData.Core/ODataStreamItem.cs

```diff
@@ -0,0 +1,81 @@
+//---------------------------------------------------------------------
+// <copyright file="ODataStreamItem.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData
+{
+    #region Namespaces
+    using Edm;
+    #endregion
+
+    /// <summary>
+    /// A class to represent a stream value
+    /// </summary>
+    public sealed class ODataStreamItem : ODataItem
+    {
+        /// <summary>Primitive Type, if known.  Either String, Binary, or None.</summary>
+        private EdmPrimitiveTypeKind typeKind;
+
+        /// <summary>
+        /// Creates an ODataStreamValue with a specified typeKind.
+        /// </summary>
+        /// <param name="typeKind">The type of stream value.</param>
+        public ODataStreamItem(EdmPrimitiveTypeKind typeKind)
+        {
+            this.PrimitiveTypeKind = typeKind;
+        }

```

Comment:
Should we call the second Constructor? 

Or, this one is useless?

Reply:
In most cases, there will not be a content type.  Fixed to have this one call the second with a null content type.

---
In reply to: [268357951](https://github.com/OData/odata.net/pull/1204#discussion_r268357951) [](ancestors = 268357951)